### DISCUSSION TITLE
DAB Radio build 20230310

### DISCUSSION THREAD

**Discussion by gmarin37** [rern/rAudio#1274] [Bug Reports]
Hi, my dab radio dongle works now on a runeaudi on raspberry 4/8go 
![coverart13](https://user-images.githubusercontent.com/3191979/227318237-3477e8bb-fe7d-41be-8dc5-ad18c530af71.jpg)
I see 17075 stations on the library but when I clic on DAB Radio the page is blank : no listing or images...
What can I check to make it works completely?
Regards
Gilbert

**Comment by gmarin37** [comment]
ok I have created the directory /srv/http/data/dabradio for user/group http. now I can do a scan and I see some stations but when I want to listen one I'am redirected to the screen playlist

**Reply by hjfisch** [reply]
Thank's a lot . This was the solution for me . Finally I've got it working now with 61 stations in Germany .
